plugins/ps/functions

```diff
@@ -194,7 +194,16 @@ ps_scale() {
   local DOKKU_PROCFILE="$DOKKU_ROOT/$APP/DOKKU_PROCFILE"
   shift 1
 
-  [[ ! -e $DOKKU_SCALE_FILE ]] && generate_scale_file "$APP" "$IMAGE_TAG" >/dev/null 2>&1
+  [[ ! -e $DOKKU_PROCFILE ]] && extract_procfile "$APP" "$IMAGE_TAG"
```

Comment:
Might want to append `>/dev/null 2>&1 || true` to the end of this so we don't get any errors when the app isn't deployed yet and doesn't have a built image.

Reply:
👍

The only case I can see where this will fail is if there are errors in the `Procfile`, but we don't want to fail in those cases either.